examples/README.md

```diff
@@ -8,85 +8,159 @@ title: Examples
 
 # Runme Examples
 
-This `README.md` contains examples for running automated e2e tests for this extension.
+Runme enables you to execute interactive runbooks using Markdown. More specifically, Runme runs your code and commands inside your fenced code blocks (shell, bash, zsh, but also, Ruby, Python, etc) see [shebang.md](shebang.md) to see more examples using different programming languages.
 
-# Extension Example Markdown Files
+## Shell
```

Comment:
maybe we can have a short cell to run here... before we get to the massive example below.

at first, i thought the example was too big but now I actually like it. however, i think we need a short executable "ice breaker" here